### PR TITLE
Start of readme

### PR DESCRIPTION
**Pull request by perploug**
Issue: #5 - Update Readme

## Description
We need a helpful and friendly readme of what the OST repo is about, what it will contain and the purpose of the open source team. 

## Types of Changes
_What types of changes does your code introduce? Put an `x` in all the boxes that apply:_
- Documentation / non-code

## Tasks
  - [x] Add purpose of the open source team
  - [x] Add description of what the repository will contain
  - [x] Add Contribution notice
  - [x] Add contact information

## Checklist
_Go over all the following points, and put an `x` in all the boxes that apply:_
- [ ] My change requires a change to the documentation & I have updated it accordingly.

### CODE REVIEW

**Comment by perploug** [comment]
PR Ready for review #50

File: README.md [hunk 1]
```diff
@@ -1,2 +1,46 @@
 # Zalando Open Source Team 
 
+The repository and issue tracker of the Zalando Open Source Team (Contact: opensource@zalando.de). Includes the team website and issue tracker with the tasks we are currently working on.
```
Comment (princiya) [inline README.md:3]: issue tracker is mentioned twice here, can we combine this into one line?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,46 @@
 # Zalando Open Source Team 
 
+The repository and issue tracker of the Zalando Open Source Team (Contact: opensource@zalando.de). Includes the team website and issue tracker with the tasks we are currently working on. 
+
+opensource.zalando.net
```
Comment (princiya) [inline README.md:5]: best to include the whole url: http[s]://opensource.zalando.net

File: README.md [hunk 3]
```diff
@@ -1,2 +1,46 @@
...
+
+
+## The purpose of the open source team
```
Comment (princiya) [inline README.md:8]: best to title case 'Open Source' in all occurrences .

File: README.md [hunk 4]
```diff
@@ -1,2 +1,46 @@
...
+The Zalando Open Source Team is responsible for nurturing and growing open source and open practices at Zalando. 
+We work across departments as internal consultants to remove barriers for contributing, producing and consuming open
+source code. 
+
+We advice and assist on:
+
+- Creating and nurturing open source communities around Zalando Open Source projects
+- Align community interests with Zalando business interests
+- Efficient and compliant adobtion of open source code by Zalando teams
```
Comment (princiya) [inline README.md:17]: typo: adoption

File: README.md [hunk 5]
```diff
@@ -1,2 +1,46 @@
...
+- Recruiting from open source communities 
+- Best practices for applying open source to internal projects
+
+We are also here to demystify and simply answer questions, so no matter if you are working at Zalando or not, get in touch: opensource@zalando.de
```
Comment (princiya) [inline README.md:21]: 😮 demystify 🎊

File: README.md [hunk 6]
```diff
@@ -1,2 +1,46 @@
...
+
+## Purpose of this repository
+This repository primary serves as the public website, issue tracker and history of the Open Source Team. We do however plan to share additional resources and tooling in the future via this repository.
+
+## Have a look at what we are working on 
+
+- Our project backlog is public, and can be found under the [projects page](https://github.com/zalando/ost/projects/1) 
+- The Issue tracker is also open to view and add issues to [Issue tracker](https://github.com/zalando/ost/issues)
```
Comment (princiya) [inline README.md:29]: title case 'Issue Tracker' or not?

File: SECURITY.md [hunk 7]
```diff
@@ -0,0 +1 @@
+If you have discovered a security vulnerability, please email tech-security@zalando.de
```
Comment (princiya) [inline SECURITY.md:1]: security isn't our headache 😆

**Review by princiya — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -1,2 +1,46 @@
...
+If you have general questions around open source @ zalando, the quickest way to get in touch is via email: opensource@zalando.de.
+
+## Contributing
+
+Please read [CONTRIBUTING.md] for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Team members
+
+* **Paul Adams** - paul.adams@zalando.de - [@therealpadams](https://github.com/therealpadams)
+* **Per Ploug** - per.ploug@zalando.de - [@pploug](https://github.com/perploug)
```
Comment (princiya) [inline README.md:40]: Is your username `pploug` or `perploug`? :)

File: SECURITY.md (hunk 7 above)
Comment (perploug) [reply SECURITY.md:1]: This is the standard security.md file recommended by rules of play :)

File: README.md (hunk 5 above)
Comment (perploug) [reply README.md:21]: (verb) - make (a difficult subject) clearer and easier to understand  :)

**Comment by perploug** [comment]
Managed to delete my own fork, opened new PR containing the requested changes #53

**Pull request closed without merging**